packages/route/tests/Application/Kernel.php

```diff
@@ -41,13 +46,26 @@ public function registerBundles(): iterable
     {
         $bundles = [...parent::registerBundles()];
 
+        $hasDynamicRouting = false;
         foreach ($bundles as $key => $bundle) {
             // remove old route bundle to avoid conflicts
             if (DeprecatedSuluRouteBundle::class === $bundle::class
                 || SuluPreviewBundle::class === $bundle::class
+                || \str_contains($bundle::class, 'Sulu')
+                || \str_contains($bundle::class, 'Massive')
+                || \str_contains($bundle::class, 'PHPCR')
+                || \str_contains($bundle::class, 'SecurityBundle')
```

Comment:
currently having lot of conflicts with Sulu and other bundles as that combinations not work properly so I removed all bundles which are not required.